### PR TITLE
N1 Nice to Have

### PR DESCRIPTION
**Pull request by suryanshkushwaha**
- Added the texting feature in the collab view for users to text at the same time

### CODE REVIEW

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx [hunk 1]
```diff
@@ -120,14 +122,13 @@ const CollaborationServiceIntegratedView: React.FC = () => {
 
   const handleLangChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
     setLanguage(parseInt(e.target.value));
-    setSyntaxFullLang(e.target.textContent!);
```
Comment (sp4ce-cowboy) [inline peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:123]: This code is important, it needs to be in the codebase in order to convey the information about the programming language used. setSyntaxLang only deals with judge0 compatible numbers.

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx [hunk 2]
```diff
@@ -120,14 +122,13 @@ const CollaborationServiceIntegratedView: React.FC = () => {
...
-    setSyntaxLang(e.target.value === '63' ? 'javascript'
```
Comment (sp4ce-cowboy) [inline peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:124]: Are these changes necessary? We would preferably want to minimize unnecessary formatting changes - they affect both code contributions and unduely change the formatting that we are already familiar with, let alone accidental changes that might affect code functionality (such as the above comment).

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx [hunk 3]
```diff
@@ -262,42 +263,53 @@ const CollaborationServiceIntegratedView: React.FC = () => {
...
-                mode: syntaxLang,
-                lineNumbers: true,
-                tabSize: 2,
-                indentWithTabs: true,
-                showHint: true,
-                extraKeys: {
-                    'Ctrl-Space': 'autocomplete', // Trigger autocomplete with Ctrl-Space
-                },
-                hintOptions: { completeSingle: false },
+              mode: syntaxLang,
```
Comment (sp4ce-cowboy) [inline peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:267]: Same as above.

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx [hunk 4]
```diff
@@ -262,42 +263,53 @@ const CollaborationServiceIntegratedView: React.FC = () => {
...
+              indentWithTabs: true,
+              showHint: true,
+              extraKeys: {
+                'Ctrl-Space': 'autocomplete', // Trigger autocomplete with Ctrl-Space
+              },
+              hintOptions: { completeSingle: false },
             }}
 
             editorDidMount={(editor) => {
-                editor.on('keyup', (cm: any, event: any) => {
```
Comment (sp4ce-cowboy) [inline peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:282]: Same as above

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx [hunk 5]
```diff
@@ -262,42 +263,53 @@ const CollaborationServiceIntegratedView: React.FC = () => {
...
-            />
+          />
         </div>
-      
+        {sessionId &&<Chat sessionId={sessionId} />}
+      </div>
+
       <h3 style={{ textAlign: 'left', marginBottom: '5px' }}>Output</h3>
-      <div className="output-container" style={{ width: '900px', textAlign: 'left', border: '1px solid #ddd', padding: '10px', borderRadius: '5px', backgroundColor: '#f9f9f9', overflowY: 'scroll' }}>
+      <div className="output-container" style={{ width: '100%', textAlign: 'left', border: '1px solid #ddd', padding: '10px', borderRadius: '5px', backgroundColor: '#f9f9f9' }}>
```
Comment (sp4ce-cowboy) [inline peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:299]: Same as above. Removing overFlowY will prevent the console from scrolling, and expand the console onto the page

**Review by sp4ce-cowboy — CHANGES_REQUESTED** [review]
Some comments about code reformatting and some critical changes to be reverted.

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx (hunk 1 above)
Comment (suryanshkushwaha) [reply peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:123]: this was probably a mistake during merge, resolved

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx (hunk 2 above)
Comment (suryanshkushwaha) [reply peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:124]: this line is unchanged, unsure why it shows as removed

File: peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx (hunk 3 above)
Comment (suryanshkushwaha) [reply peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx:267]: that's my vs code formatting in place as per the 2103T guidelines
